### PR TITLE
running new icons through react-svgr

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Ran recent icons through https://react-svgr.com/playground/?native=true per @jeanregisser 's suggestion

### Tested

Visually inspected that the icons still looked 💯

### CODE REVIEW

File: packages/mobile/src/icons/VerificationComplete.tsx [hunk 1]
```diff
@@ -3,14 +3,14 @@ import Svg, { Path, Circle } from 'svgs'
 
 function VerificationComplete() {
   return (
-    <Svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (MuckT) [inline packages/mobile/src/icons/VerificationComplete.tsx:6]: Add a `testID="VerificationComplete"`

File: packages/mobile/src/icons/VerificationDenied.tsx [hunk 2]
```diff
@@ -3,22 +3,15 @@ import Svg, { Path, Circle } from 'svgs'
 
 function VerificationDenied() {
   return (
-    <Svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <Circle cx="30" cy="30" r="30" fill="#F9F6F0" />
-      <Circle cx="30" cy="30" r="11.25" stroke="#2E3338" stroke-width="1.5" />
+    <Svg width={60} height={60} fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (MuckT) [inline packages/mobile/src/icons/VerificationDenied.tsx:6]: Add `testID="VerificationDenied"`

File: packages/mobile/src/icons/VerificationPending.tsx [hunk 3]
```diff
@@ -3,16 +3,16 @@ import Svg, { Path, Circle } from 'svgs'
 
 function VerificationPending() {
   return (
-    <Svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <Circle cx="30" cy="30" r="30" fill="#F9F6F0" />
+    <Svg width={60} height={60} fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (MuckT) [inline packages/mobile/src/icons/VerificationPending.tsx:6]: Add `testID="VerificationPending"`

**Review by MuckT — CHANGES_REQUESTED** [review]
Just a couple of testID's to add! LGTM.

File: packages/mobile/src/icons/VerificationComplete.tsx (hunk 1 above)
Comment (jeanregisser) [reply packages/mobile/src/icons/VerificationComplete.tsx:6]: How about `testID="VerificationCompleteIcon"` instead? to be clear it's an icon.

**Review by jeanregisser — APPROVED** [review]
Thanks! 👍 
Already approved! But I agree with @MuckT about adding `testID` for easier e2e testing.
Would be great to suffix them with `Icon` too.

**Review by MuckT — APPROVED** [review]

**Pull request merged**